framework/packages/abstract-core/src/core/manager.rs

```diff
@@ -180,14 +188,23 @@ pub enum ExecuteMsg {
         // Provide list of module to install after sub-account creation
         install_modules: Vec<(ModuleInfo, Option<Binary>)>,
     },
+    /// Unregister sub-account
```

Comment:
Small suggestion (idk if we want to do this) but instead of having all the sub-account actions on the top level we could create a new `enum SubAccountAction` and have a single variant here `SubAccount(SubAccountAction)`. 

It would screw over cw-orch's function generation tho :/

Reply:
Sure, I like it more! And by the way, it will get fixed in the next cw-orch version :rocket: 
https://github.com/AbstractSDK/cw-orchestrator/pull/196/commits/0cafc1c176b0adb7b249c9c93399ced90aac3a2a

Reply:
Yeah but it will require us to construct the inner `SubAccountAction` type to do the call.